diff --git a/app/controllers/articles_controller.rb b/app/controllers/articles_controller.rb
index f10d546..2dfeaf6 100644
--- a/app/controllers/articles_controller.rb
+++ b/app/controllers/articles_controller.rb
@@ -70,6 +70,18 @@ class ArticlesController < ApplicationController
     end
   end
 
+  def action_of_checkbox
+    # @articles = Article.all
+    # @markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+    #
+    # if params[:search]
+    #   @articles = Article.search(params[:search]).order('created_at DESC')
+    # else
+    #   @articles = Article.all.order('created_at DESC')
+    # end
+    # render action: 'index'
+  end
+
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_article
diff --git a/app/views/articles/index.html.erb b/app/views/articles/index.html.erb
index 2d511bb..cecbf19 100644
--- a/app/views/articles/index.html.erb
+++ b/app/views/articles/index.html.erb
@@ -32,6 +32,11 @@
         <td><%= link_to 'Show', article, class: "btn btn-info" %></td>
         <% if user_signed_in? %>
           <% if article.user_id == current_user.id %>
+            <td>
+              <a href="<%= article_action_of_checkbox_path(article) %>">
+                <input type='checkbox'></input>
+              </a>
+            </td>
             <td><%= link_to 'Edit', edit_article_path(article), class: "btn btn-warning" %></td>
             <td><%= link_to 'Destroy', article, class: "btn btn-danger", method: :delete, data: { confirm: "Do you want to delete the articles #{article.title}?" } %></td
           <% end %>
diff --git a/config/routes.rb b/config/routes.rb
index 27cf387..552c788 100644
--- a/config/routes.rb
+++ b/config/routes.rb
@@ -2,6 +2,7 @@ Rails.application.routes.draw do
   devise_for :users
   get '/users/sign_out', controller: "devise/sessions", action: "destroy"
   resources :articles do
+    get :action_of_checkbox
     resources :comments
   end
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
